frontend/src/lib/components/TaxonomicFilter/InfiniteList.tsx

```diff
@@ -60,20 +60,48 @@ const staleIndicator = (parsedLastSeen: dayjs.Dayjs | null): JSX.Element => {
     )
 }
 
+const unusedIndicator = (eventNames: string[]): JSX.Element => {
```

Comment:
UI/UX wise I think we might be better off showing the property name muted and then adding an info icon instead with the tooltip information. The "unused" tag pulls too much attention and contrary to stale events, there's nothing to fix here.

Reply:
Something like this?

![2021-12-17 14 59 17](https://user-images.githubusercontent.com/53387/146555543-8c331f79-e9cb-446d-857c-9b9559204105.gif)

Reply:
Disagree - I think clear labeling is better than the info icon in this case. Also those options are selectable - showing them in gray makes them look disabled. We need to inform, but not bias the list. Simply saying it's unseen and specifying a range of time seems like enough to me.

Reply:
The label does seem clearer. It'd make sense to make the unusable events muted at the same time I think.